pkg/config/config_test.go

```diff
@@ -82,6 +82,12 @@ var _ = Describe("Configuration", func() {
 					RangeEnd:   "02:00:00:00:00:00:00:FF"},
 				SubnetManager: SubnetManagerPluginConfig{Plugin: "noop"}}
 
+			os.Setenv("KUBERNETES_SERVICE_HOST", "10.0.0.1")
+			os.Setenv("KUBERNETES_SERVICE_PORT", "443")
+			defer func() {
+				os.Unsetenv("KUBERNETES_SERVICE_HOST")
+				os.Unsetenv("KUBERNETES_SERVICE_PORT")
+			}()
 			err := dc.ValidateConfig()
```

Comment:
how does it not fail on the rest of the ValidateConfig calls ?
KUBERNETES_SERVICE_HOST/PORT is not (supposed) to be defined and also KUBE_CONFIG_FILE is not defined in the BeforeEach clause